lib/ansible/modules/cloud/amazon/cloudformation_stack_set.py

```diff
@@ -0,0 +1,664 @@
+    type: bool
+    default: false
+  wait_timeout:
+    description:
+    - How long to wait (in seconds) for stacks to complete create/update/delete operations.
+    default: 900
+  capabilities:
+    description:
+    - Capabilities allow stacks to create and modify IAM resources, which may include adding users or roles.
+    - Currently the only available values are 'CAPABILITY_IAM' and 'CAPABILITY_NAMED_IAM'. Either or both may be provided.
+    - >
+        The following resources require that one or both of these parameters is specified: AWS::IAM::AccessKey,
+        AWS::IAM::Group, AWS::IAM::InstanceProfile, AWS::IAM::Policy, AWS::IAM::Role, AWS::IAM::User, AWS::IAM::UserToGroupAddition
+    choices:
+    - 'CAPABILITY_IAM'
+    - 'CAPABILITY_NAMED_IAM'
+  regions:
+    description:
+    - A list of AWS regions to create instances of a stack in. The `region` parameter chooses where the Stack Set is created, and `regions`
+      specifies the region for stack instances.
+    - At least one region must be specified to create a stack set. On updates, if fewer regions are specified only the specified regions will
+      have their stack instances updated.
+  accounts:
+    description:
+    - A list of AWS accounts in which to create instance of CloudFormation stacks.
+    - At least one region must be specified to create a stack set. On updates, if fewer regions are specified only the specified regions will
+      have their stack instances updated.
+  administration_role_arn:
+    description:
+    - ARN of the administration role, meaning the role that CloudFormation Stack Sets use to assume the roles in your child accounts.
+    aliases:
+    - admin_role_arn
+    - admin_role
+    - administration_role
+  execution_role_name:
+    description:
+    - ARN of the execution role, meaning the role that CloudFormation Stack Sets assumes in your child accounts.
+    - This MUST NOT be an ARN, and the roles must exist in each child account specified.
+    aliases:
+    - exec_role_name
+    - exec_role
+    - execution_role
+  tags:
+    description:
+      - Dictionary of tags to associate with stack and its resources during stack creation. Can be updated later, updating tags removes previous entries.
+  failure_tolerance:
+    description:
+    - Settings to change what is considered "failed" when running stack instance updates, and how many to do at a time.
+
+author: "Ryan Scott Brown (@ryansb)"
+extends_documentation_fragment:
+- aws
+- ec2
+requirements: [ boto3>=1.6, botocore>=1.8 ]
+'''
+
+EXAMPLES = '''
+- name: Create a stack set with instances in two accounts
+  cloudformation_stack_set:
+    name: my_stack
+    description: Test stack in two accounts
+    state: present
+    template_url: https://s3.amazonaws.com/my-bucket/cloudformation.template
+    accounts: [1234567890, 2345678901]
+    regions:
+    - us-east-1
+
+- name: on subsequent calls, templates are optional but parameters and tags can be altered
```

Comment:
When I create a stackset it succeeds but isn't showing changed=True for the first playbook run. When I run the identical playbook again subsequently I'm consistently getting:
```
The full traceback is:
Traceback (most recent call last):
  File "/var/folders/by/k8_fbl593dlctgqmwq5wzl2c0000gn/T/ansible_AgmmVT/ansible_module_cloudformation_stack_set.py", line 664, in <module>
    main()
  File "/var/folders/by/k8_fbl593dlctgqmwq5wzl2c0000gn/T/ansible_AgmmVT/ansible_module_cloudformation_stack_set.py", line 552, in main
    OperationId=operation_ids[-1],
  File "/var/folders/by/k8_fbl593dlctgqmwq5wzl2c0000gn/T/ansible_AgmmVT/ansible_modlib.zip/ansible/module_utils/aws/core.py", line 244, in deciding_wrapper
  File "/Users/shertel/Workspace/ansible/venv/python2.7.13/lib/python2.7/site-packages/botocore/client.py", line 314, in _api_call
    return self._make_api_call(operation_name, kwargs)
  File "/Users/shertel/Workspace/ansible/venv/python2.7.13/lib/python2.7/site-packages/botocore/client.py", line 612, in _make_api_call
    raise error_class(parsed_response, operation_name)
botocore.errorfactory.OperationInProgressException: An error occurred (OperationInProgressException) when calling the UpdateStackInstances operation: Another Operation on StackSet arn:aws:cloudformation:us-east-1:448830907657:stackset/my-stack:ecda1d99-1f6a-414e-b350-98051ff76f77 is in progress
```
even though checking in the console each update succeeds and there aren't any operations in progress. The only way to run the playbook without appearing to fail is setting wait: true.

Reproducer task:
```
      cloudformation_stack_set:
        administration_role_arn: "{{ admin_role_arn }}"
        execution_role_name: "{{ exec_role }}"
        name: my-stack
        description: Test stack
        state: present
        template_url: https://s3-us-west-2.amazonaws.com/cloudformation-templates-us-west-2/VPC_Single_Instance_In_Subnet.template
        accounts:
          - "{{ account }}"
        regions:
          - us-east-1
        parameters:
          KeyName: "{{ key }}"
          InstanceType: t2.micro
```